common/config/azure-pipelines/jobs/docs-build.yaml

```diff
@@ -26,6 +26,9 @@ parameters:
   - name: useCurrentTransformerDocsArtifact
     type: boolean
     default: false
+  - name: useAuthClientsDocsArtifact
```

Comment:
should this follow the pattern where the name is `useCurrent...DocArtifact`?

Reply:
Sloppy thanks